lib/private/message_settings_update.php

```diff
@@ -70,9 +70,29 @@ function generateQueryStringText( $CurrentValues, &$BindValues, $ValueName, $New
     
         return '';
     }
-    
+
     // -----------------------------------------------------------------------------
-    
+
+    function getRequestValue( $aRequest, $sParameterName, $sValueType = "IntValue" )
+    {
+        if (is_array($aRequest) && array_key_exists($sParameterName, $aRequest))
+        {
+            return $sParameterName;
```

Comment:
Sure, $aRequest[$sParameterName]. Sorry for that

You mean just 
$Settings['PurgeRaids']['IntValue']      = intval($aRequest['purgeTime']);
?? That wouldn't catch the PHP notice if the index "purgeTime" is missing in $aRequest.